frontend/src/components/FileBrowser/FileBrowser.js

```diff
@@ -40,10 +40,16 @@ export class FileBrowser extends React.Component<
         this.state = {
             currentWorkingDirectory: '',
             fileBrowserData: {},
-            isVisible: false,
+            isVisible: !this.props.startCollapsed,
         };
     }
 
+    componentDidMount() {
+        if (!this.props.startCollapsed) {
+            this.updateFileBrowser('');
+        }
+    }
+
     componentWillReceiveProps(nextProps) {
```

Comment:
This life cycle function (https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops) is actually going to be deprecated as well, maybe it would be good to refactor and remove this function as well in this PR

Reply:
Removed commponentWillReceiveProps.